doc/nrf/releases/release-notes-2.4.2.rst

```diff
@@ -0,0 +1,100 @@
+.. _ncs_release_notes_242:
+
+|NCS| v2.4.2 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF52, nRF53, nRF70, and nRF91 Series.
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+This minor release adds changes on top of the :ref:`nRF Connect SDK v2.4.1 <ncs_release_notes_241>`
```

Comment:
is this `minor` release or `bugfix` release?

Reply:
Actually a patch release, if our docs are correct https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases_and_maturity/developing/code_base.html#versions-and-revisions